### ISSUE TITLE
Is there any way to make it stop sending me an email daily?

### ISSUE THREAD

**Issue by mstevens** [rackerlabs/recap#35]
Is there any way to make it stop sending me an email daily?

I just want the logs.

**Comment by carlgeorge** [comment]
Yes, just unset the MAILTO variable in the config file.

https://github.com/rackerlabs/recap/blob/0.9.8/recap.conf#L16-L18
https://github.com/rackerlabs/recap/blob/0.9.8/recap#L718-L721

**Comment by mstevens** [comment]
Tested this, I still get emailed every day by recaplog:

[DD] packing fdisk...
[DD] do compress old log files
cat: /var/log/recap/fdisk_20150909-*.log: No such file or directory
ls: cannot access /var/log/recap/fdisk_20150909-*.log: No such file or directory
[DD] packing mysql...
[DD] do compress old log files
[DD] deleting: /var/log/recap/mysql_20150909-000001.log
[DD] deleting: /var/log/recap/mysql_20150909-001001.log
[DD] deleting: /var/log/recap/mysql_20150909-002001.log
[DD] deleting: /var/log/recap/mysql_20150909-003001.log
[DD] deleting: /var/log/recap/mysql_20150909-004001.log
[DD] deleting: /var/log/recap/mysql_20150909-005002.log
[DD] deleting: /var/log/recap/mysql_20150909-010001.log
[DD] deleting: /var/log/recap/mysql_20150909-011002.log
[DD] deleting: /var/log/recap/mysql_20150909-012001.log
[DD] deleting: /var/log/recap/mysql_20150909-013001.log
[DD] deleting: /var/log/recap/mysql_20150909-014002.log
[DD] deleting: /var/log/recap/mysql_20150909-015001.log
[DD] deleting: /var/log/recap/mysql_20150909-020001.log
[DD] deleting: /var/log/recap/mysql_20150909-021001.log
[DD] deleting: /var/log/recap/mysql_20150909-022002.log
[DD] deleting: /var/log/recap/mysql_20150909-023002.log
[DD] deleting: /var/log/recap/mysql_20150909-024001.log
[DD] deleting: /var/log/recap/mysql_20150909-025001.log
.... and continues

On further inspection it's due to cron output rather than being an email directly generated by recap.

**Comment by carlwgeorge** [comment]
Sorry @mstevens, somehow I never noticed your follow up comment.

Does changing [EMAIL_ON_ERROR to 0 in the recaplog file](https://github.com/rackerlabs/recap/blob/0.9.14/src/recaplog#L41) make a difference?  It could be that or the use of the tee command [here](https://github.com/rackerlabs/recap/blob/0.9.14/src/recaplog#L62).

**Comment by njohansson** [comment]
I can confirm setting EMAIL_ON_ERROR to 0 doesn't make any difference (just tested on our server). It shouldn't really either as you can see the check for EMAIL_ON_ERROR happens after passing message to tee in print_msg()

**Comment by StafDehat** [comment]
Seems to me that the "correct" fix would be to modify the recaplog script to accept --quiet, --silent, or --debug (and default to quiet) arguments.  The script src/recap.cron would then have to invoke recaplog with the correct arguments.

Alternatively, just edit src/recap.cron to include the typical cron "2>&1 >/dev/null".  That being said, the way we're currently running 'find' is causing output to STDERR, so we'd have to either address the 'find' invocation first, or go with a more heavy-handed "&>/dev/null":
```
# recaplog 2>&1 >/dev/null
cat: '/var/log/recap/fdisk_20161103-*.log': No such file or directory
ls: cannot access '/var/log/recap/fdisk_20161103-*.log': No such file or directory
```

**Comment by tonyskapunk** [comment]
First of all, sorry for closing that PR from @StafDehat it happened when closing #20, the fix for this problem is now proposed in #72

**Comment by tonyskapunk** [comment]
Fixed in #72
